torchrl/modules/tensordict_module/common.py

```diff
@@ -235,6 +247,14 @@ def __init__(
 
     @property
     def is_functional(self):
+        if not _has_functorch:
```

Comment:
let's make sure that we also cover cases where the user has explicitly chosen to use the native implementation